examples/heat_equation.cpp

```diff
@@ -130,9 +130,8 @@ int main()
             x_domain_begin(x_domain.front(), x_post_ghost.extents());
     // our zone at the end of the domain that will be mirrored to the
     // ghost
-    DiscreteDomain const x_domain_end(
-            x_domain.back() - x_pre_ghost.extents(),
-            x_pre_ghost.extents());
+    DiscreteDomain const
+            x_domain_end(x_domain.back(), x_pre_ghost.extents());
```

Comment:
```suggestion
            x_domain_end(x_domain.back() - x_pre_ghost.extents() + 1, x_pre_ghost.extents());
```

Reply:
These two elements only cancel themselves for size 1 ghosts

Reply:
Thx